### PR TITLE
updating information related to customer-admin-cluster role in ARO

### PR DESCRIPTION
**Pull request by Makdaam**
On hold until the feature is released.
/hold

waiting for PR https://github.com/openshift/openshift-azure/pull/1776
and the release process (new images, new version release, deployment in Azure)

I will cancel the hold after the feature is public.

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@xltian can someone from QE ack this?

**Comment by mgahagan73** [comment]
ACK, I was having issues getting the changed doc to render properly but I think it's an issue on my end. 

/lgtm

**Comment by Makdaam** [comment]
/hold cancel
the new version with documented features is rolling out to production.

**Review by mgahagan73 — APPROVED** [review]
I reviewed this doc change back in July, it still looks good to me.

**Review by huffmanca — CHANGES_REQUESTED** [review]
I have one nitpick comment in the content itself; however, with this restructure the `Cluster-level Permissions` table in the openshift-aro admin guide is missing references. Specifically, it needs the `manage_rbac` and `manage_scc` files included.

To include these files, you'll need to update the `_topic_map.yml` in the root directory, and add `openshift-aro` to the comma-separated `Distros` list underneath each file.

Let me know if you have any questions on this process.

I can confirm that the links on the index page work after adding in these files and building locally. If these files shouldn't be included in the openshift-aro distro, then you'll need to reintroduce the specific `Cluster-level Permissions` section in the openshift-aro distribution.

File: admin_guide/manage_rbac.adoc [hunk 1]
```diff
@@ -19,8 +29,8 @@ resources] and the administrator CLI to manage the
 xref:../architecture/additional_concepts/authorization.adoc#architecture-additional-concepts-authorization[roles
 and bindings].
 
-ifdef::openshift-dedicated[]
-Dedicated administrators can view but not manage cluster roles. They can manage cluster role bindings
+ifdef::openshift-dedicated,openshift-aro[]
+{cluster-admin-name} can view but not manage cluster roles. They can manage cluster role bindings
```
[the remark this answers is not in the document]
Comment (Makdaam) [reply admin_guide/manage_rbac.adoc]: FIxed.

**Comment by Makdaam** [comment]
@huffmanca,
thank you for the quick response.
Fixed the singular and added manage_scc and manage_rbac to the openshift-aro distro.

**Comment by huffmanca** [comment]
All of these changes look good to me! Merging and cherry picking.

**Pull request merged**